pkg/scheduler/framework/plugins/volumezone/volume_zone.go

```diff
@@ -158,7 +158,7 @@ func (pl *VolumeZone) Filter(ctx context.Context, _ *framework.CycleState, pod *
 			nodeV, _ := nodeConstraints[k]
 			volumeVSet, err := volumehelpers.LabelZonesToSet(v)
 			if err != nil {
-				klog.Warningf("Failed to parse label for %q: %q. Ignoring the label. err=%v. ", k, v, err)
+				klog.ErrorS(err,"Failed to parse label, ignoring the label", k, v)
```

Comment:
I think info here is enough. And `"label", fmt.Printf("%q: %q",k,v)` should be more reasonable.

Reply:
Thanks @gavinfish , it's `fmt.Sprintf`. fixed.